src/ui/TeamAxisTab.py

```
@@ -0,0 +1,508 @@
+        axis = get_axis_class(self.selected_axis_id())
+        signature = self.fixed_team_signature()
+
+        if axis is None:
+            return False, og.app.tr("没有可用的固定轴方案")
+        if not fixed_team.get("enabled", False):
+            return False, og.app.tr("请先在“队伍管理”中启用固定队伍")
+        if len(signature) != 4 or any(key is None for key in signature):
+            return False, og.app.tr("固定队伍的四个槽位都必须选择内置角色代码")
+        if not axis.matches_signature(signature):
+            return False, og.app.tr("当前固定队伍与所选固定轴不匹配")
+        return True, og.app.tr("队伍匹配，可以启用该固定轴")
```

Ruff (0.15.18)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)